test/test_dists.jl

```diff
@@ -491,8 +497,8 @@ function test_pairwise(dist, x, y, T)
             rxx[i, j] = evaluate(dist, x[:, i], x[:, j])
         end
         # As earlier, we have small rounding errors in accumulations
-        @test pairwise(dist, x, y) ≈ rxy
-        @test pairwise(dist, x) ≈ rxx
+        @test pairwise(dist, x, y, dims=2) ≈ rxy
+        @test pairwise(dist, x, dims=2) ≈ rxx
```

Comment:
I suppose these two lines don't need change since `pairwise(dist, x, y)` still works (although with depwarn).

L500-L501 is duplicated to L502-L503

Reply:
Hm, yes, either remove, or leave to test for behavior without `dims` keyword.

Reply:
I don't see why you would want to keep something that throws a depwarn, so I removed these duplicate line.